reference_guide/intellij_artifacts.md

```diff
@@ -0,0 +1,131 @@
+There are two parts to each example: the repository and the dependency sections.
+
+#### Gradle Example for IntelliJ IDEA Community distribution
+These snippets illustrate incorporating an IntelliJ snapshot build into a build.gradle file.
+
+**Repository Section**  
+This snippet selects the snapshot repository for the artifact by virtue of the URL.
+```
+repositories {
+    maven { url "https://www.jetbrains.com/intellij-repository/snapshots" }
+}
+```
+
+**Dependencies Section**  
+Given the repository section has selected a snapshot, the dependencies section specifies the desired build artifact
+based on the Artifact Description and Artifact Version.
+```
+dependencies {
+    idea_dep group: 'com.jetbrains.intellij.idea', name: 'ideaIC', version: '141-SNAPSHOT', ext: 'zip'
+}
+```
+Where:
+* `idea_dep` names this specification as a custom build configuration.
+* `group` corresponds to the `groupId` for builds in the Artifact Description section.
+* `name` corresponds to the `artifactId` in the Artifact Description section.
+* `version` specifies the artifact version - see the Artifact Version section.
+* `ext` corresponds to the `packaging` in the Artifact Description section.
+
+@nik - Is `idea_dep` being declared as this particular artifact dependency?
+
+Here is an example of a [build.gradle](https://github.com/shalupov/idea-cloudformation/blob/9007023afa187a1fb8b45c3ca66d5a51f86b795c/build.gradle)
+that is currently in use.
+
+#### Gradle Example for an Iindividual Module from the IntelliJ IDEA project
+These snippets illustrate incorporating separate IntelliJ modules into a build.gradle file.
+
+**Repositories Section**  
+This snippet selects the release respository with the first URL, and the repository of IntelliJ IDEA dependencies with the second URL.
+The second URL is important because this example selects individual modules, not a full build. 
+```
+repositories {
+	maven { url "https://www.jetbrains.com/intellij-repository/releases" }
+	maven { url "https://jetbrains.bintray.com/intellij-third-party-dependencies" }
+}
+```
+
+**Dependencies Section**  
+This dependencies section specifies the desired module artifacts based on the Artifact Description and Artifact Version.
+```
+dependencies {
+	compile "com.jetbrains.intellij.platform:jps-model-serialization:182.2949.4"
+	compile "com.jetbrains.intellij.platform:jps-model-impl:182.2949.4"
+}
+```
+Where:
+* `com.jetbrains.intellij.platform` corresponds to the `groupId` for modules in the Artifact Description section.
+* `jps-model-serialization` corresponds to the `artifactId` in the Artifact Description section.
+* `182.2949.4` specifies the artifact version - see the Artifact Version section.
+* `jps-model-impl` corresponds to the module `artifactId` in the Artifact Description section.
+* **Note:** the artifact version must match in both statements.
+
+@nik - Are the statements related in terms of both specifying `jps-model-`? Is there an example where only one module
```

Comment:
Yes, these two artifacts are related, `jps-model-serialization` declares some API and `jps-model-impl` provides its implementation. Currently if you want to use classes from `jps-model-serialization` you also need to manually add dependency on `jps-model-impl`. Probably we'll get rid of this later and dependency will be added automatically.

As an example with only one artifact you can use `util` artifact from the same `com.jetbrains.intellij.platform` group.